python/paddle/trainer_config_helpers/layers.py

```diff
@@ -5658,7 +5660,7 @@ def lambda_cost(input,
                           than the size of a list, the algorithm will sort the
                           entire list of get gradient.
     :type max_sort_size: int
-    :param name: The name of this layers. It is not necessary.
+    :param name: The name of this layer. It is not necessary.
```

Comment:
please change the sentence "It is not necessary." into "It is not required."
The same below.